assets/src/eejs/utils/is-schema.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * External imports
+ */
+import { isPlainObject, isUndefined, lowerCase } from 'lodash';
+
+/**
+ * Returns whether the given value is a schema response object from a schema
+ * request.
+ *
+ * @param {Object} response
+ * @return {boolean} true if this is a schema response object.
+ */
+export const isSchemaResponse = ( response ) => {
+	return (
+		hasSchemaProperty( response ) &&
+		isSchema( response.schema )
+	);
+};
+
+/**
+ * Returns whether the given value is a schema object.
+ *
+ * @param {Object} schema
+ * @return {boolean} true if this is a schema object.
+ */
+export const isSchema = ( schema ) => {
+	return (
+		isPlainObject( schema ) &&
+		! isUndefined( schema.$schema ) &&
```

Comment:
`$schema` ???  is that `$` supposed to be in there?

Reply:
yes it is.  The `$schema` property indicates what draft of the json-schema.org spec is being followed.